crtauth/rsa.py

```diff
@@ -177,7 +182,13 @@ def _str_to_int(data):
     the octets as an arbitrarily sized unsigned integer in network byte
     order
     """
-    return int(data.encode('hex'), 16)
+
+    if six.PY3:
```

Comment:
Let's switch to binascii.hexlify for both cases (I'm guessing that it works based on the availability of binascii.hexlify in my python 2.7 environment)